src/main/java/it/gov/pagopa/pu/debtpositions/connector/organization/config/OrganizationApisHolder.java

```diff
@@ -0,0 +1,47 @@
+package it.gov.pagopa.pu.debtpositions.connector.organization.config;
+
+import it.gov.pagopa.pu.organization.client.generated.OrganizationSearchControllerApi;
+import it.gov.pagopa.pu.organization.generated.ApiClient;
+import it.gov.pagopa.pu.organization.generated.BaseApi;
+import jakarta.annotation.PreDestroy;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+@Lazy
+@Service
+public class OrganizationApisHolder {
+
+    private final OrganizationSearchControllerApi organizationSearchControllerApi;
+
+    private final ThreadLocal<String> bearerTokenHolder = new ThreadLocal<>();
+
+    public OrganizationApisHolder(
+            @Value("${rest.organization.base-url}") String baseUrl,
```

Comment:
add this property on application.yml, it's value should be configurable through environment variable.
in helm folder, let's give to it a value (see p4pa-workflow-worker for the right value)